hindiCG.txt

```diff
@@ -0,0 +1,72 @@
+ DELIMITERS = "<$।>" ;
+ SUBREADINGS = LTR ;
+
+LIST POST = post;
+LIST MADJ = (adj m);
+LIST NOUN = n;
+LIST FADJ = (adj f);
+LIST NOM = (n nom);
+LIST FN = (n f );
+LIST MN = (n m );
+LIST VBSER = vbser;
+LIST CNJ = cnjcoo;
+LIST ADJ = adj;
+LIST OBL = (n obl);
+LIST ADV = adv;
+
+SECTION
+
+#1 
+SELECT ADV IF 
+	(0 ADJ )
+	(1 ADJ );
+#2
+SELECT ADJ IF 
+	(0 ADJ ) 
+	(0 NOUN ) 
+	(1 NOUN );
+#3
+SELECT FADJ 
+	(1 FN );
+
+#4
+SELECT MADJ  IF
+	(1 MN );
+
+#5
+SELECT NOM IF
+	(1 ADJ ) 
+	(2 VBSER );
+
+#6
+SELECT OBL IF 
+	(1 CNJ )
+ 	(2 NOUN ) 
+	(3 POST );
+
+#7
+SELECT OBL IF 
+	(1 POST );
+
+#8
+SELECT FADJ IF
+	(-1 FN );
+
+#9
+SELECT MADJ IF 
+	(-1 MN );
+
+#10
+SELECT NOM IF
+	(1 VBSER);
+
+#11
+SELECT (adj pl) IF 
+	(-3 NOUN)
+	(-2 CNJ)
+	(-1 NOUN)
+	(1 VBSER);
```

Comment:
Would it be better if you selected plural adjectives when the verbs are plural too? The example you gave has "लडकी आैर लडका विशाल है" but really the है should have अनुस्वार on top, right, so that will indicate the plurality better than checking just the previous positions (what if it is a longer noun phrase?).

Reply:
I agree. I'll change that . But I guess it'll be a new rule as the verbs themselves don't have a plurality rule.

Reply:
Wouldn't an adjective preceding a verb be kind of rare in Hindi? I'll add a rule nonetheless.

I'll also add a rule determining plurality of verbs using the है + अनुस्वार form

Reply:
Added these rules